rust/clients/bridge/Cargo.toml

```diff
@@ -0,0 +1,24 @@
+[package]
+name = "bridge"
```

Comment:
The name should be more explanatory, e.g. `qaul-bridge` or `qaul-matrix-bridge`. IMHO `qaul-matrix-bridge` would be best, as it is the most self-explanatory.